server/manager/src/main/java/org/apache/accumulo/manager/state/MergeStats.java

```diff
@@ -136,17 +128,10 @@ public MergeState nextMergeState(AccumuloClient accumuloClient, CurrentState man
         log.info("Merge range is already contained in a single tablet {}", info.getExtent());
         state = MergeState.COMPLETE;
       } else if (hosted == total) {
-        if (info.isDelete()) {
-          if (!lowerSplit) {
-            log.info("Waiting for {} lower split to occur {}", info, info.getExtent());
-          } else if (!upperSplit) {
-            log.info("Waiting for {} upper split to occur {}", info, info.getExtent());
-          } else {
-            state = MergeState.WAITING_FOR_CHOPPED;
-          }
-        } else {
-          state = MergeState.WAITING_FOR_CHOPPED;
-        }
+        // Todo: Clean up references to WAITING_FOR_CHOPPED and SPLITTING and remove
+        // from enum in a future PR as both are going away. for now just change
```

Comment:
See my other comment about serialization of the ordinals, and the risks of removing these enums.